framework/processor/src/main/java/com/google/android/catalog/framework/processor/SampleProcessor.kt

```diff
@@ -169,9 +173,11 @@ class ${sampleFile}Module {
         return CatalogSample(
             "$sampleName",
             "$sampleDescription",
+            listOf(${sampleTags.joinToString(",") { "\"$it\"" }}),
```

Comment:
Maybe I am missing something but why not to just use `sampleTags.toList()`?

Reply:
Using `sampleTags.toList()` make type error in generated codes as below

<img width="661" alt="image" src="https://user-images.githubusercontent.com/7759511/209189849-12dfd2b5-deb0-4668-bb72-1c5591571ed1.png">

Reply:
Oh yeah, ofc because it's not pass as a param.